systems/ukama-agent/node-gateway/Int.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM 003664043471.dkr.ecr.us-east-1.amazonaws.com/e2e-base-image:latest
+
+ENV PROJECT_NAME=api-gateway
```

Comment:
shouldn't the project name be `node-gateway` ?

Reply:
fixed